package.json

```diff
@@ -4,15 +4,15 @@
   "description": "A Blackbaud Sky API/UX demo application.",
   "main": "index.js",
   "scripts": {
-    "build": "grunt build",
     "setup": "node ./bin/setup.js",
+    "dev": "nodemon ./bin/start.js",
     "start": "node ./bin/start.js",
     "lint": "node ./node_modules/.bin/eslint server",
     "test": "npm run lint",
-    "heroku-postbuild": "echo Skip build on Heroku"
+    "build": "cd skyux-spa-ui && npm i && npm i -g @skyux-sdk/cli@next && skyux build"
```

Comment:
```suggestion
    "build": "cd skyux-spa-ui && npm i && npx -p @skyux-sdk/cli skyux build"
```

Reply:
☝️ You can use `npx` to run a package globally (without installing it on the system). This is good practice for running global scripts on a CI.